### PR TITLE
fix: 2 typos

### PR DESCRIPTION
**Pull request by RoboSchmied**
fix typo: endianess -> endianness

### CODE REVIEW

File: depends/packages/expat.mk [hunk 1]
```diff
@@ -6,7 +6,7 @@ $(package)_sha256_hash=f79b8f904b749e3e0d20afeadecf8249c55b2e32d4ebb089ae378df47
 
 # -D_DEFAULT_SOURCE defines __USE_MISC, which exposes additional
 # definitions in endian.h, which are required for a working
-# endianess check in configure when building with -flto.
+# endianness check in configure when building with -flto.
```
Comment (UdjinM6) [inline depends/packages/expat.mk:9]: this one is inherited from bitcoin, fixing it here can potentially cause merge conflicts in the future

File: src/dashbls/depends/relic/include/relic_util.h [hunk 2]
```diff
@@ -253,7 +253,7 @@
 /*============================================================================*/
 
 /**
- * Toggle endianess of a digit.
+ * Toggle endianness of a digit.
```
Comment (UdjinM6) [inline src/dashbls/depends/relic/include/relic_util.h:256]: same reason but this time the origin is relic lib

**Review by UdjinM6 — CHANGES_REQUESTED** [review]
NACK, see below

**Comment by PastaPastaPasta** [comment]
Thanks for you contribution; but I agree with Udjin. We shouldn't fix these here to avoid merge conflicts in the future.

File: depends/packages/expat.mk (hunk 1 above)
Comment (RoboSchmied) [reply depends/packages/expat.mk:9]: Thank you and sorry for not noticing that. I am going to fix it upstream.

File: src/dashbls/depends/relic/include/relic_util.h (hunk 2 above)
Comment (RoboSchmied) [reply src/dashbls/depends/relic/include/relic_util.h:256]: Thank you and sorry for not noticing that. I am going to fix it upstream.

**Pull request closed without merging**